bedrock/bedrock-common/src/main/java/com/nukkitx/protocol/bedrock/data/LevelEventType.java

```diff
@@ -74,9 +74,9 @@
     GLOBAL_PAUSE, //data: 1 to pause, 0 to resume
     SIM_TIME_STEP,
     SIM_TIME_SCALE,
-    ACTIVATE_BLOCK,
 
     // blocks
+    ACTIVATE_BLOCK,
```

Comment:
Not sure why you moved this. It's a world event even though it says block in the name.